front/lib/api/tracker.ts

```diff
@@ -0,0 +1,89 @@
+import type { TrackerConfigurationType } from "@dust-tt/types";
+
+import { sendEmailWithTemplate } from "@app/lib/api/email";
+
+const TRACKER_FROM_EMAIL = "dev@dust.tt";
+const TRACKER_FROM_NAME = "Bob Tracker"; // 😬
+
+export async function sendTrackerEmailWithNoGeneration(
+  tracker: TrackerConfigurationType
+): Promise<void> {
+  const recipients = new Set(tracker.recipients);
+  if (!recipients.size) {
+    throw new Error("No recipients found for tracker");
+  }
+
+  const _sendTrackerEmailWithNoGenerationToRecipient = async (
+    recipient: string,
+    tracker: TrackerConfigurationType
+  ): Promise<void> => {
+    await sendEmailWithTemplate({
+      to: recipient,
+      from: {
+        name: TRACKER_FROM_NAME,
+        email: TRACKER_FROM_EMAIL,
+      },
+      subject: `[Dust] Tracker ${tracker.name} check complete: No updates required.`,
+      body: `
+        <p>Tracker: ${tracker.name}.</p>
+        <p>No changes detected in watched documents. All maintained documents are current</p>
+      `,
+    });
+  };
+
+  await Promise.all(
```

Comment:
`to` takes an array no ? Instead of promise.all we could potentially send to several recipients at once ?

Reply:
Nope I expected that too but takes a string, and it calls sendEmail who also takes a string and I decided not to refactor

Reply:
Ah too bad, would have been cool to have a thread where you can reply all with your other coworkers

Reply:
You can do that for free if you set your recipient to a google group! 
(But yes, we could refactor to send only one email to multiple recipient, ok to do it after we get our v0 running?)